nucypher/__init__.py

```diff
@@ -24,10 +24,3 @@
 __all__ = [
     "__title__", "__summary__", "__version__", "__author__", "__license__", "__copyright__", "__email__", "__url__"
 ]
-
-
-# Set Default Curve #
-#####################
-
-from umbral.config import set_default_curve
-set_default_curve()
```

Comment:
Good to see this removed.  This is not needed any more since `umbral >= 0.2` only uses a single curve?

Reply:
Yep, no need to worry about curves or parameter objects anymore.